controller/config.go

```diff
@@ -1062,7 +1065,15 @@ func (c Config) SkipConfirmation() bool {
 	if v, ok := c[SkipConfirmation]; ok {
 		return v.(bool)
 	}
-	return DefaultSkipConfirmation
+	if environmentValue := os.Getenv(osenv.JujuSkipConfirmationEnvKey); environmentValue != "" {
+		boolValue, err := strconv.ParseBool(environmentValue)
+		if err != nil {
+			errors.Errorf("Value (%s) of JUJU_SKIP_CONFIRMATION env var is not a boolean", osenv.JujuSkipConfirmationEnvKey)
```

Comment:
Instead of using the value of JUJU_SKIP_CONFIRMATION, personally I would ignore the value and just look if it's set or not. Set => skip confirmation, unset => don't skip

That will make this less rigid. But will mean the case of JUJU_SKIP_CONFIRMATION=false will count as true. What do we do elsewhere>